tools/cloud-build/daily-tests/ansible_playbooks/slurm-integration-test.yml

```diff
@@ -89,7 +84,10 @@
       ansible.builtin.set_fact:
         login_ip: "{{ get_login_ip.stdout }}"
       when: '"*" in login_node'
-
+    - name: Check that login IP is set
+      ansible.builtin.assert:
+        that:
+        - login_ip != ""
```

Comment:
I'm new to Ansible, so I'm not 100% sure, but I think an empty string is different than a undefined variable.

Reply:
I am not 100% either but, since you what have is explicit and clear I am fine to leave it. I guess you could have 
```
      ansible.builtin.assert:	     
        that:	       
        - login_ip is defined	
        - login_ip != ""	
```

Do as you please and close this comment.